### PR TITLE
[BEAM-2141] Updates jenkins job for JDBCIOIT

### PR DESCRIPTION
**Pull request by chamikaramj**
This is a slightly updated version of Stephen Sisk's https://github.com/apache/beam/pull/3604.

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
run seed job

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run JDBC Performance Test

**Comment by aaltay** [comment]
@chamikaramj is this PR still needed?

**Comment by chamikaramj** [comment]
Yes, this is blocked till https://issues.apache.org/jira/browse/INFRA-14819 is resolved.

**Comment by chamikaramj** [comment]
Run seed job

**Comment by chamikaramj** [comment]
Run Standalone Seed Job

File: .test-infra/jenkins/common_job_properties.groovy [hunk 1]
```diff
@@ -254,20 +254,26 @@ class common_job_properties {
...
-        // Clone appropriate perfkit branch
-        shell('git clone https://github.com/GoogleCloudPlatform/PerfKitBenchmarker.git')
-        // Install Perfkit benchmark requirements.
-        shell('pip install --user -r PerfKitBenchmarker/requirements.txt')
-        // Install job requirements for Python SDK.
-        shell('pip install --user -e sdks/python/[gcp,test]')
-        // Launch performance test.
-        shell("python PerfKitBenchmarker/pkb.py $pkbArgs")
+      // Launch performance test.
+      shell("python PerfKitBenchmarker/pkb.py $pkbArgs")
```
Comment (kennknowles) [inline .test-infra/jenkins/common_job_properties.groovy]: I don't know that you can have multiple `steps` sections, can you? I think this is one of the things with this DSL.

Comment (chamikaramj) [reply .test-infra/jenkins/common_job_properties.groovy]: This script used to work few weeks back. Did something change ?

Comment (kennknowles) [reply .test-infra/jenkins/common_job_properties.groovy]: This PR seems to change it. The other failure is unrelated, but implies you are building not against the merge commit, but against the head of the PR. That seems like a bug itself in our Jenkins triggering, but you might just try rebasing.

**Comment by chamikaramj** [comment]
Run seed job

**Comment by reuvenlax** [comment]
What's the status of this PR?

**Comment by chamikaramj** [comment]
I pinged https://issues.apache.org/jira/browse/INFRA-14819. This PR needs to have kubectl installed in Beam Jenkins executors and currently only executor beam1 has it.

**Pull request closed without merging**